sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -250,6 +250,26 @@ def test_one_hot_encoder_handle_unknown():
     assert_raises(ValueError, oh.fit, X)
 
 
+def test_one_hot_encoder_handle_unknown_strings():
+    X = np.array(['11111111', '22', '333', '4444']).reshape((-1, 1))
+    X2 = np.array(['55555', '22']).reshape((-1, 1))
+
+    # Test the ignore option, when categories are strings
+    # particularly when the known category strings are larger
+    # than the unknown category strings
```

Comment:
I think this remains a bit obscure to the reader, and you should reference the issue number ("See #xxxx")